queue/adaptive_order_storage.go

```diff
@@ -85,6 +85,11 @@ func (items *adaptiveOrderItems) updateWaiting(ctx context.Context) {
 			continue
 		}
 
+		if ti.DispatchBy.After(now) {
```

Comment:
Use `IsStale`

Reply:
I want to use the same "now" value, hence this one being weird.

Reply:
You could make a parameterized helper for `IsStale`.